packages/presentation/lib/src/pages/login_page/login.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/generated/l10n.dart';
+import 'package:presentation/src/base_bloc/base_tile.dart';
+import 'package:presentation/src/base_bloc/bloc_screen.dart';
+import 'package:presentation/src/navigation/base_page.dart';
+import 'package:presentation/src/pages/login_page/widgets/form_widget.dart';
+import 'package:presentation/style/dimens.dart';
+import 'package:presentation/style/text_styles/styles.dart';
+
+import 'bloc/login_bloc.dart';
+import 'bloc/login_data.dart';
+
+class Login extends StatefulWidget {
+  const Login({super.key});
+
+  static const _routeName = '/LoginPage';
+
+  static page() => BasePage(
+        key: const ValueKey(_routeName),
+        name: _routeName,
+        builder: (context) => const Login(),
+        isShowNavBar: true,
+      );
+
+  @override
+  State<Login> createState() => _LoginState();
+}
+
+class _LoginState extends BlocScreenState<Login, LoginBloc> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
```

Comment:
since the AppBar logic is the same everywhere I think we can move it to the App's level as we did it with bottom navigation

Reply:
not everywhere, on details screen we have another appBar which have a lot of differences with home and login. it seems difficult, and Egor says that he convinced u to change your mind =)